django-os2webscanner/os2webscanner/models/scannerjobs/exchangescanner_model.py

```diff
@@ -41,8 +41,7 @@ def get_absolute_url(self):
         return '/exchangescanners/'
 
     def create_scan(self):
-        from .exchangescan_model import ExchangeScan
-        return ExchangeScan.create(self)
+        return super().create_scan()
```

Comment:
This function can presumably now be deleted, since it just calls the superclass implementation.